ee/modules/380-metallb/openapi/config-values.yaml

```diff
@@ -0,0 +1,152 @@
+              type: string
+            operator:
+              type: string
+            tolerationSeconds:
+              type: integer
+              format: int64
+            value:
+              type: string
+  bgpPeers:
+    type: array
+    default: []
+    description: |
+      A list of external BGP routers to use with the module.
+
+      Format — a data array similar to that of [MetalLB's](https://metallb.universe.tf/configuration/#bgp-configuration).
+      The parameter is optional if only the `layer2` mode is used to announce services.
+    items:
+      type: object
+      required: ["peer-address", "peer-asn", "my-asn"]
+      properties:
+        peer-address:
+          type: string
+          description: |
+            The IP address of the external BGP router.
+          x-examples: [ "192.168.42.15" ]
+          pattern: '^([0-9]{1,3}\.){3}[0-9]{1,3}$'
+        peer-asn:
+          type: integer
+          description: |
+            The AS number on the external BGP router.
+          x-examples: [ "65535" ]
+        my-asn:
+          type: integer
+          description: |
+            The AS number in the cluster.
+          x-examples: [ "65535" ]
+        source-address:
+          type: string
+          description: |
+            Source IP address for outbound connections.
+          x-examples: [ "192.168.42.15" ]
+          pattern: '^(?:[0-9]{1,3}\.){3}[0-9]{1,3}$'
+        hold-time:
+          anyOf:
+            - type: integer
+            - type: string
+          x-kubernetes-int-or-string: true
+          pattern: '^(0|(([0-9]+)y)?(([0-9]+)w)?(([0-9]+)d)?(([0-9]+)h)?(([0-9]+)m)?(([0-9]+)s)?(([0-9]+)ms)?)$'
+          x-examples: [ "5s", "150" ]
+          description: |
+            The timeout after which the neighboring BGP peer is considered lost. This value is divided by three to get the keep-alive interval.
+
+            The recommended value is `3s` (i.e., keep-alive packets are sent once per second). Note that the BGP protocol does not support lower values.
+            By default, the parameter is set to `90s` (i.e., keep-alive packets are sent every 30 seconds).
+        node-selector:
+          type: object
+          additionalProperties:
+            type: string
+          description: |
+            The additional pseudo-selector implemented by the speaker application. It selects nodes allowed to connect to external BGP routers. Do not confuse it with `speaker.nodeSelector` and  `nodeSelector`.
+
+            An optional parameter.
+            The format is [`matchLabels` or `matchExpressions`](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements).
+  addressPools:
+    type: array
+    default: []
+    description: |
+      A list of IP ranges to assign to services.
+
+      Format — a data array similar to that of [MetalLB's](https://metallb.universe.tf/configuration/#advanced-address-pool-configuration).
+    items:
```

Comment:
It would be nice to have a common x-examples with two types of pools. It is convenient to copy&paste.

Reply:
good point, done